src/content.ts

```diff
@@ -0,0 +1,72 @@
+const fetchData = () => {
+  const emoteData = [];
+  fetch('https://api.betterttv.net/3/cached/emotes/global', {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+    .then((response) => response.json())
```

Comment:
IIRC you can give the `.json()` method a type, so this could be 
```ts
response.json<{id: string; code: string; imageType: string; userId: string}>();
```
which will let us clean the code up a bit a couple lines down